docs/core/tools/dotnet-format.md

```diff
@@ -0,0 +1,146 @@
+---
+title: dotnet format command
+description: The dotnet format command formats code to match `editorconfig` settings for the current directory.
+ms.date: 07/12/2021
+---
+# dotnet format
+
+**This article applies to:** ✔️ .NET 6.x SDK and later versions
+
+## Name
+
+`dotnet format` - Formats code to match `editorconfig` settings.
+
+## Synopsis
+
+```dotnetcli
+dotnet format [options] [<PROJECT | SOLUTION>]
+
+dotnet clean -h|--help
+```
+
+## Description
+
+`dotnet format` is a code formatter that applies style preferences to a project or solution. Preferences will be read from an .`editorconfig` file, if present, otherwise a default set of preferences will be used. For more information on `editorconfig` files see the documentation [here](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/configuration-files#`editorconfig`).
```

Comment:
"here" should be avoided as link text. Also it may be better to use relative link.